README.md

```diff
@@ -9,31 +9,25 @@
 
 ## 🚀 Overview
 
-The **Avo Inspector Adobe Tags Extension** allows you to **monitor and validate** your **Adobe Client Data Layer** by sending **only data schemas** (no personal data) to Avo Inspector. This helps you ensure data consistency and catch schema violations in real time.
+The **Avo Inspector Adobe Tags Extension** allows you to **monitor and validate** your data schemas by sending **only data structures** (no personal data) to Avo Inspector. This helps you ensure data consistency and catch schema violations in real time.
 
 ✅ **The integration does NOT send personal data to Avo Inspector; only the structures of the data is sent.**  
 ✅ This means your **user data remains secure** while allowing you to monitor the health of your tracking implementation.
 
-#### [Avo docucementation](https://www.avo.app/docs/inspector/connect-inspector-to-adobe-tag-extension)
+#### [Avo documentation](https://www.avo.app/docs/inspector/connect-inspector-to-adobe-tag-extension)
```

Comment:
I assume we need to update some docs alongside this?

Reply:
https://github.com/avohq/docs/pull/1484